src/main/java/core/basesyntax/util/PropertyReader.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyReader {
+    private static final Properties properties = new Properties();
+
+    static {
+        try (InputStream inputStream = PropertyReader.class
+                .getClassLoader()
+                .getResourceAsStream("application.properties")) {
+            if (inputStream != null) {
+                properties.load(inputStream);
+            } else {
+                throw new RuntimeException("Sorry, unable to find application.properties");
```

Comment:
It's a good practice to avoid hardcoding file names. Consider passing the file name as a parameter to the method or class.

Reply:
the logic was changed